x-pack/plugins/enterprise_search/public/applications/app_search/components/documents/search_experience/search_experience.tsx

```diff
@@ -21,6 +21,9 @@ import { externalUrl } from '../../../../shared/enterprise_search_url';
 import { SearchBoxView, SortingView } from './views';
 import { SearchExperienceContent } from './search_experience_content';
 import { buildSearchUIConfig } from './build_search_ui_config';
+import { CustomizationCallout } from './customization_callout';
+import { CustomizationModal } from './customization_modal';
+import { useLocalStorage } from '../../../../shared/use_local_storage';
```

Comment:
nit: move `import { useLocalStorage }` to between line 19 and 20, we should try to group external vs internal/shared vs component-specific imports together

Reply:
75ed8eb8097bede056126295ee0758202567bd0d